<?xml version='1.0' encoding='ISO-8859-1' ?>
<!DOCTYPE requirement SYSTEM "requirement.dtd">

<requirement id='engine.get.no' type='BUG' prio='LOW' status="CLOSED">
   <topic>Accessing an unknown message in xmlBlaster with the synchronous get() method throws an Exception</topic>
   <description>-</description>
   <example lang="Java">
      <![CDATA[
      try {
         String xmlKey = "<key oid='NotExistingMessage' queryType='EXACT'></key>";
         String qos = "<qos></qos>";
         MessageUnitContainer[] msgArr = xmlBlaster.get(xmlKey, qos);
      } catch(XmlBlasterException e) {
         System.out.println("Success, caught Exception for trying to get unknown message: "+e.reason);
      }
      ]]>
   </example>
   <author>ruff@swand.lake.de</author>
   <date>2000 02 23</date>
   <revision>$Revision: 1.9 $</revision>
   <testcase status="CLOSED">
      <name>-</name>
      <comment>-</comment>
      <test tool="SUITE">org.xmlBlaster.test.qos.TestGet</test>
   </testcase>
</requirement>

